### ISSUE TITLE
Single author doesn't work properly in word

### ISSUE THREAD

**Issue by Benjamin-Lee** [pandoc-scholar/pandoc-scholar#33]
With one author listed in the metadata, the Word output has an extra `, and` before the author's name.

For example:
<img width="162" alt="image" src="https://user-images.githubusercontent.com/13618205/56087894-7d076580-5e42-11e9-8887-75e4fd9bdc6e.png">

When the author field is listed as:

```
author:
  - Benjamin D. Lee:
      institute:
        - Lab41
        - HMS
        - SEAS
      email: benjamin_lee@college.harvard.edu
      orcid: 0000-0002-7133-8397
      correspondence: "yes"
```

**Comment by tarleb** [comment]
Thanks for the report! I have fixed the problem and will release a new version within the next days.

**Comment by tarleb** [comment]
Should be fixed in v2.1.1.
